src/pages/App.tsx

```diff
@@ -89,6 +91,16 @@ export default function App() {
     }, 1000)
   }, [])
 
+  const stacklyPopup = useStacklyPopup()
+  const [closedStacklyPopup] = useUpdateClosedStacklyPopup()
+  const showStacklyPopup = process.env.REACT_APP_SHOW_STACKLY_POPUP === 'true'
+
+  useEffect(() => {
+    if (closedStacklyPopup || !showStacklyPopup) return
+    const id = stacklyPopup()
+    return () => toast.done(id)
+  }, [closedStacklyPopup, showStacklyPopup, stacklyPopup])
```

Comment:
Once you move it outside the component. The use effect doesn't show that as a dependency. 
```suggestion
  }, [closedStacklyPopup, stacklyPopup])
```